src/ray/raylet/scheduling/cluster_resource_scheduler.h

```diff
@@ -201,6 +201,13 @@ class ClusterResourceScheduler {
   /// Get number of nodes in the cluster.
   int64_t NumNodes();
 
+  /// Add an (unused) local resource.
+  ///
+  /// \param resource_name: Resource which we want to update.
+  /// \param resource_total: New capacity of the resource.
+  void AddLocalResource(
```

Comment:
Why dont' we call it `AddcustomResources?`

Reply:
Oh I think this would work for adding a predefined resource, but local is the important property because it avoids lookups and sets resource instances.